### DISCUSSION TITLE
How to Enter Kindle Paperwhite Serial Number in DeDRM settings

### DISCUSSION THREAD

**Discussion by profjg** [noDRM/DeDRM_tools#311] [Q&A] [answered]
This may be posted somewhere but I cannot find instructions on how to add Kindle Paperwhite serial number to DeDRM settings. Hoping this route will help me remove DRM from new Kindle books purchased from Amazon. I can still remove DRM with older books.

**Comment by ElleKayEm** [comment, marked as answer]
In calibre, Preferences > Plugins.  Select DeDRM under File Type plugins.  Click Customize plugin > Kindle eInk ebooks > green plus sign.  Easiest way to get the serial number correct is copy and paste from the website.  Click OK and Apply on your way back out of the dialog boxes.  Restart calibre before adding your books.

**Comment by profjg** [comment]
Thank you!

**Reply by profjg** [reply]
Worked perfectly!
